src/libsystemd/meson.build

```diff
@@ -101,9 +101,11 @@ libsystemd_static = static_library(
         libsystemd_sources,
         install : false,
         include_directories : includes,
-        link_with : libbasic,
+        link_with : [libbasic,
+                     libbasic_openssl],
```

Comment:
I think it should be possible to tell meson that `libbasic_openssl` uses `openssl`, and then it will not be necessary to specify `libopenssl` separately in `dependencies`.